/learning: add Pluto section 🎈

fonsp opened the pull request:
Hi all!

This PR is some self-promotion, I would like to mention Pluto in the `/learning/` page. 🙈 We are super passionate about improving the beginner experience and I think the project has reached a point where it is valuable to recommend Pluto.jl to beginner users!

<img width="995" alt="Scherm­afbeelding 2024-01-17 om 12 47 38" src="https://github.com/JuliaLang/www.julialang.org/assets/6933510/ffe6fb65-80f8-4981-825d-32f6f5ba40e2">

I added some styles to make the new section look nice and attractive and to give it a "Pluto look". I can remove this if you like, but we find that beginner Julia users value our modern design (e.g. this [experience talk](https://youtu.be/S-2aLXV03gM?list=FLf4UMPLT4Xap3-qXOIJ6OKg&t=90)), and I like to think that it makes Julia look more accessible/cool/modern.

The section includes three random screenshots of Pluto, which link to https://featured.plutojl.org/. If people are interested in seeing Julia and Pluto in action, this is the place to go! And we think it's also a good learning resource for Julia.

LilithHafner commented on learning/index.md:
```diff
@@ -114,6 +114,43 @@
+      <div class="col-lg-4 col-md-3 language-features"><hr/></div>
+    </div>
+
+    <p><a href="https://plutojl.org/">Pluto.jl</a> is a Julia programming environment designed for <em>learning and teaching</em>, and it is the easiest way to get started with Julia programming, packages and visualisation.</p>
```
```suggestion
    <p><a href="https://plutojl.org/">Pluto.jl</a> is a Julia programming environment designed for <em>learning and teaching</em>, and it is a great way to get started with Julia programming, packages and visualisation.</p>
```
For large demographics, the easiest way to start with Julia is either pasting the juliaup command or using a replit service, so I'm not comfortable with the unqualified claim of "easiest"

LilithHafner reviewed (commented):
Pluto seems worth advertising here. 

Some image nits:

- The aspect ratio of the three images are different, that's not great, aesthetically.
- 1450 pixels wide may be excessively large when the display style on large browsers is three-across, however, it should be Franklin's responsibility to serve appropriately sized images, and I'm assuming it will down-sample as needed.
- Does Pluto.jl support .svg screenshots?

LilithHafner commented:
It would also be nice of the screenshots linked to the specific featured notebooks they are from

fonsp commented:
Hey @LilithHafner thanks for the review!

I updated my PR:
- New screenshots with consistent aspect ratio and lower DPI (can you compress the PNG files again or tell me how to do it? Pluto does not support svg screenshots)
- Now all three screenshots come from an actual featured notebook, and I added links and alttext for all three

LilithHafner commented:
For pluto8.png I applied a 256-color quantization first before lossless compression. This shouldn't have had a visible impact for that image because it is grayscale with 2 extra colors, but I used `pngnq` which apparently chose to pick a slightly different shade of red. I think that's okay.
